spec/spec_helper.rb

```diff
@@ -51,6 +51,16 @@
   # triggering implicit auto-inclusion in groups with matching metadata.
   config.shared_context_metadata_behavior = :apply_to_host_groups
 
+  # Many RSpec users commonly either run the entire suite or an individual
+  # file, and it's useful to allow more verbose output when running an
+  # individual spec file.
+  if config.files_to_run.one?
```

Comment:
@Paulmulonzia - try the following command to see what this change did: 

`rake spec SPEC=spec/collectionspace/converter/publicart/acquisition_spec.rb`

I prefer this test output when I'm running tests on a single file or example group within a file, as it shows me a full report of what my code is expected to do. I also try to write my code and tests so this output can serve as quick documentation of the code's function. 

If this change causes a problem for you, let me know. I can force this to run for myself on the command line when I run tests if it's a problem.

Reply:
@kspurgin, is this a replacement of the current test command:

> rspec spec/collectionspace/converter/publicart/acquisition_spec.rb

Reply:
Not a replacement, but a different way to run one test file.

Works exactly the same. 

`rspec spec/collectionspace/converter/publicart/acquisition_spec.rb`

Will give you fuller, more detailed test output for the tests in the file---both passing and failing 

`rspec` will run all the tests and report details only on failures.

Reply:
Great. Sounds good.